Uncaught Error: Cannot find module 'crypto'

```
Uncaught Error: Cannot find module 'crypto'
    at require (modules-runtime.js?hash=0969a31…:133)
    at meteorInstall.node_modules.steem.lib.auth.index.js (modules.js?hash=2f1f987…:6398)
    at fileEvaluate (modules-runtime.js?hash=0969a31…:207)
    at require (modules-runtime.js?hash=0969a31…:130)
    at meteorInstall.node_modules.steem.index.js (modules.js?hash=2f1f987…:5267)
    at fileEvaluate (modules-runtime.js?hash=0969a31…:207)
    at Module.require (modules-runtime.js?hash=0969a31…:130)
    at Module.Mp.import (modules.js?hash=2f1f987…:374)
    at meteorInstall.imports.api.steem.js (steem.js:1)
    at fileEvaluate (modules-runtime.js?hash=0969a31…:207)
    at Module.require (modules-runtime.js?hash=0969a31…:130)
    at Module.Mp.import (modules.js?hash=2f1f987…:374)
    at meteorInstall.imports.api.collections.userSettings.js (userSettings.js:1)
    at fileEvaluate (modules-runtime.js?hash=0969a31…:207)
    at Module.require (modules-runtime.js?hash=0969a31…:130)
    at Module.Mp.import (modules.js?hash=2f1f987…:374)
```

Version:
```
    "steem": "^0.4.8",
```

Related Issue:
https://github.com/steemit/steem-js/issues/39

--- a/package.json
+++ b/package.json
@@ -34,7 +34,6 @@
     "bluebird": "^3.4.6",
     "bs58": "^4.0.0",
     "bytebuffer": "^5.0.1",
-    "crypto": "0.0.3",
     "debug": "^2.2.0",
     "detect-node": "^2.0.3",
     "ecurve": "^1.0.5",

